src/extension.ts

```diff
@@ -193,6 +199,13 @@ export async function activate(context: vscode.ExtensionContext) {
             '"'
         )
 
+        if (isDifferentVersion(context)) {
```

Comment:
Consider putting this block in its own method, and have the method eat+log errors to reduce the potential for breaking extension activation.